DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/README.md

```diff
@@ -17,20 +17,50 @@ This DPC++ reference design demonstrates a highly optimized image sensor adaptiv
 ## Purpose
 This FPGA reference design demonstrates a parameterizable image processing pipeline that implements an Adaptive Noise Reduction (ANR) algorithm using a bilateral filter. See the [Additional Design Information Section](#additional-design-information) for more information on the ANR algorithm itself and how it was implemented for the FPGA.
 
-## License  
+## License
 Code samples are licensed under the MIT license. See
 [License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
 
+### Using Visual Studio Code*  (Optional)
+
+You can use Visual Studio Code (VS Code) extensions to set your environment,
+create launch configurations, and browse and download samples.
+
+The basic steps to build and run a sample using VS Code include:
+ - Download a sample using the extension **Code Sample Browser for Intel oneAPI Toolkits**.
+ - Configure the oneAPI environment with the extension **Environment Configurator for Intel oneAPI Toolkits**.
+ - Open a Terminal in VS Code (**Terminal>New Terminal**).
+ - Run the sample in the VS Code terminal using the instructions below.
+ - (Linux only) Debug your GPU application with GDB for Intel® oneAPI toolkits using the Generate Launch Configurations extension.
+
+To learn more about the extensions, see
+[Using Visual Studio Code with Intel® oneAPI Toolkits](https://software.intel.com/content/www/us/en/develop/documentation/using-vs-code-with-intel-oneapi/top.html).
+
+After learning how to use the extensions for Intel oneAPI Toolkits, return to this readme for instructions on how to build and run a sample.
+
 ## Building the Reference Design
 
+> **Note**: If you have not already done so, set up your CLI
```

Comment:
Do these instructions about the CLI apply in the Dev Cloud?  If not we should probably add a note to that effect.

I'm actually not sure this note belongs here at all, especially at this level of detail.  Perhaps we could refer to an external document about how to set up your environment properly instead?  Possibly specifically noting 'see the section on setvars to make sure your CLI (defnintion of CLI goes here...) is set up properly'

Reply:
The DX team requested to have some sort of setvars instructions included in the sample README. In too many user studies, a user will click on directions for setvars that takes them to another page (say, the Base Kit Get Started). Then, even though the link says click here only if you don't know how to source setvars, and return back to this README after you have sourced it,  they continue to follow directions in the new document (BaseKit GSG). So DX asked me to reduce the possibility of users jumping off page. That is why we have a minimal set of instructions here.
But you do bring up a good point about DevCloud. The placement of the setvars instructions above the DevCloud instructions suggests that setvars is required for DevCloud and it is not (at least I'm pretty sure). There are setvars instructions on the DevCloud Get Started instructions so that probably has to be changed as well. I added https://jira.devtools.intel.com/browse/DOC-9774 to fix this, and targeted the JIRA for 2022.3.
Thanks, @mdbtucker taking the time to review!